pretix_eth/exporter.py

```diff
@@ -0,0 +1,116 @@
+from collections import OrderedDict
+
+from django import forms
+
+from pretix.base.exporter import ListExporter
+from pretix.base.models import (
+    OrderPayment,
+    OrderRefund,
+)
+from pretix_eth.models import WalletAddress
+
+import pytz
+
+
+class EthereumOrdersExporter(ListExporter):
+    identifier = 'ethorders'
+    verbose_name = 'Ethereum orders and refunds'
+
+    @property
+    def additional_form_fields(self):
+        return OrderedDict(
+            [
+                ('payment_states',
+                 forms.MultipleChoiceField(
+                     label='Payment states',
+                     choices=OrderPayment.PAYMENT_STATES,
+                     initial=[OrderPayment.PAYMENT_STATE_CONFIRMED,
+                              OrderPayment.PAYMENT_STATE_REFUNDED],
+                     widget=forms.CheckboxSelectMultiple,
+                     required=False
+                 )),
+                ('refund_states',
+                 forms.MultipleChoiceField(
+                     label='Refund states',
+                     choices=OrderRefund.REFUND_STATES,
+                     initial=[OrderRefund.REFUND_STATE_DONE, OrderRefund.REFUND_STATE_CREATED,
+                              OrderRefund.REFUND_STATE_TRANSIT],
+                     widget=forms.CheckboxSelectMultiple,
+                     required=False
+                 )),
+            ]
+        )
+
+    def iterate_list(self, form_data):
+        payments = OrderPayment.objects.filter(
+            order__event__in=self.events,
+            state__in=form_data.get('payment_states', []),
+            provider='ethereum'
+        ).order_by('created')
+
+        refunds = OrderRefund.objects.filter(
+            order__event__in=self.events,
+            state__in=form_data.get('refund_states', []),
+            provider='ethereum'
+        ).order_by('created')
+
+        objs = sorted(list(payments) + list(refunds), key=lambda obj: obj.created)
+
+        headers = [
```

Comment:
since this is a constant list of the same strings, maybe consider moving to a class or module level constant?